baseplate/frameworks/queue_consumer/kafka.py

```diff
@@ -130,25 +130,32 @@ def handle(self, message: confluent_kafka.Message) -> None:
                 span.set_tag("kafka.timestamp", message.timestamp())
 
                 blob: bytes = message.value()
-
-                try:
-                    data = self.message_unpack_fn(blob)
-                except Exception:
-                    logger.error("skipping invalid message")
-                    context.trace.incr_tag(f"{self.name}.{topic}.invalid_message")
-                    return
-
-                try:
-                    ingest_timestamp_ms = data["endpoint_timestamp"]
-                    now_ms = int(time.time() * 1000)
-                    message_latency = (now_ms - ingest_timestamp_ms) / 1000
-                except KeyError:
-                    # we can't guarantee that all publishers populate this field
-                    # v2 events publishers (event collectors) do, but future
-                    # kafka publishers may not
-                    message_latency = None
-
-                self.handler_fn(context, data, message)
+                data: Any = None
+                message_latency: Optional[int] = None
+
+                if not blob:
```

Comment:
as a stylistic nit, I'd generally prefer to put the normal case first if we're branching the whole thing. Makes it easier to read through the happy path.